examples/nextjs-with-typescript/styles.css

```diff
@@ -100,29 +100,37 @@ li a.react {
   }
 }
 
+mux-player:not([audio]),
+mux-video:not([audio]) {
+  max-height: 450px;
+}
+
 mux-player:not([audio]),
 mux-video:not([audio]),
 video {
```

Comment:
Simplified a lot of this stuff by wrapping the component (e.g. `<MuxPlayer>` + `.options` in a `<main>` element and using flex layout. Also allowed me to remove some of the JS code for sizing. Tested on mobile and desktop.